exercises/leap/example_gen.go

```diff
@@ -32,9 +32,7 @@ type js struct {
 // template applied to above data structure generates the Go test cases
 var tmpl = `package leap
 
-// Source: {{.Ori}}
-{{if .Commit}}// Commit: {{.Commit}}
-{{end}}
+{{.Header}}
```

Comment:
One thing I was talking about in https://github.com/exercism/xgo/issues/605#issuecomment-290891332 (and I should have talked about it in #604 as well): 

> If, in the course of writing a generator, we find that there is significant duplicate code between generators that could be extracted into https://github.com/exercism/xgo/blob/master/gen/gen.go, we should do so, so that writing each generator is as little work as possible.

This is one thing I was talking about - we keep writing this header and it's the same three lines every time (and it we add the version, it's going to be four lines). Since it's so common, I propose it be just one line.